client/src/components/VtkViewer.vue

```diff
@@ -255,50 +285,52 @@ export default {
         const ctx = myCanvas.getContext('2d');
         ctx.clearRect(0, 0, myCanvas.width, myCanvas.height);
 
-        const worldLines = this.convertWorldSlicesToLines();
-        const [displayLine1, displayLine2] = this.convertWorldLinesToDisplayLines(worldLines);
-        displayLine1[0][1] = myCanvas.height - displayLine1[0][1];
-        displayLine1[1][1] = myCanvas.height - displayLine1[1][1];
-        displayLine2[0][1] = myCanvas.height - displayLine2[0][1];
-        displayLine2[1][1] = myCanvas.height - displayLine2[1][1];
-        if (this.name === 'x') {
-          ctx.strokeStyle = '#b71c1c';
-          ctx.beginPath();
-          ctx.moveTo(...displayLine1[0]);
-          ctx.lineTo(...displayLine1[1]);
-          ctx.stroke();
+        if (this.showCrosshairs) {
+          const worldLines = this.convertWorldSlicesToLines();
+          const [displayLine1, displayLine2] = this.convertWorldLinesToDisplayLines(worldLines);
+          displayLine1[0][1] = myCanvas.height - displayLine1[0][1];
+          displayLine1[1][1] = myCanvas.height - displayLine1[1][1];
+          displayLine2[0][1] = myCanvas.height - displayLine2[0][1];
+          displayLine2[1][1] = myCanvas.height - displayLine2[1][1];
+          if (this.name === 'x') {
+            ctx.strokeStyle = '#b71c1c';
+            ctx.beginPath();
+            ctx.moveTo(...displayLine1[0]);
+            ctx.lineTo(...displayLine1[1]);
+            ctx.stroke();
```

Comment:
The above lines are repeated quite a lot, it's probably worth making a helper function for them.